actors/builtin/reward/reward_state.go

```diff
@@ -9,6 +9,12 @@ import (
 // A quantity of space * time (in byte-epochs) representing power committed to the network for some duration.
 type Spacetime = big.Int
 
+// 36.242212 FIL
```

Comment:
@ZX I'm having trouble confirming this.  Have not done so yet.

Reply:
https://www.wolframalpha.com/input/?i=330000000*%281-e%5E-%28Log%5B2%5D%2F%286+*+60*60*365*24%2F+30+%29%29%29

with the latest changes, 30 seconds block time and 365 days year, this should be the right value for initial position for per epoch reward